distributionviewer/core/static/css/logout.styl

```diff
@@ -1,7 +1,16 @@
 @import 'lib';
 
 .sign-out-wrapper {
+  .email {
+    display: none;
+  }
   .button {
     margin-left: $min-padding;
   }
 }
+
+@media $base-desktop {
+  .sign-out-wrapper .email {
```

Comment:
Same here, we could select just `.email` if we're not worried about the level of specificity.